app/webpacker/css/darkswarm/typography.scss

```diff
@@ -150,6 +150,8 @@ ul.check-list {
   padding-bottom: 100px;
 }
 
+.red { color: red }
```

Comment:
I'm not sure this is the best way to highlight this, but will await Rachel's feedback.